### PR TITLE
MBS-8762: JSON: Serialize the ISNIs for artist and labels.

### PR DESCRIPTION
**Pull request by Zastai**
Seemed fairly straightforward. Did not add a role (as exists for the IPIs) because AFAIK only Artists have ISNIs.

### CODE REVIEW

**Comment by Zastai** [comment]
Sigh. Looks like rebasing to limit the diffs between the branch and master was not the right thing after all.
Will see about dropping this PR and making a new, clean, one.

**Comment by Zastai** [comment]
Managed to repair it. Might get the hang of git eventually.

**Comment by Zastai** [comment]
So this new version does use a Role, as Labels have ISNIs too.

File: lib/MusicBrainz/Server/WebService/Serializer/JSON/2/Role/ISNIs.pm [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package MusicBrainz::Server::WebService::Serializer::JSON::2::Role::ISNIs;
+use Moose::Role;
+use List::UtilsBy qw( sort_by );
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/WebService/Serializer/JSON/2/Role/ISNIs.pm]: This and the line below can be removed since the imports aren't used.

**Comment by Zastai** [comment]
You mean a separate PR for the removal in IPIs.pm? Or just as a separate commit in this one for cherry-pick purposes?

**Comment by mwiencek** [comment]
You can add it as a separate commit to this PR (and I'll merge it with the PR, not cherry-pick it). But don't worry about it, I'll just merge this and fix them.

**Comment by mwiencek** [comment]
Done in https://github.com/metabrainz/musicbrainz-server/commit/056e9560507951586b4eda9eb5d47bec78d72b06

**Pull request merged**